DeleteUserDataActivity/__tests__/backupAndDelete.test.ts

```diff
@@ -37,12 +37,18 @@ const asyncIteratorOf = <T>(items: T[]): AsyncIterator<T[]> => {
       const value = data.shift();
       return {
         done: typeof value === "undefined",
-        value: [value]
+        value: [value!]
```

Comment:
Could be this unsafe? In tests I don't see particular problems.

Reply:
At the end, I preferred to rewrite the function with a simpler implementation :) See [[#IOPID-1450] refactor based on comments, part 1](https://github.com/pagopa/io-functions-admin/pull/231/commits/13289a70c86f945983ec6e17fee2ccff25ad2f77)